main/preload.js

```diff
@@ -3,6 +3,7 @@
 /** @typedef {import('electron').IpcRendererEvent} IpcRendererEvent */
 /** @typedef {import('./typings').Activity} Activity */
 /** @typedef {import('./typings').FILTransaction} FILTransaction */
+// eslint-disable-next-line max-len
```

Comment:
can you think of a way to improve these? Can typedef / type imports be written over multiple lines, or shorter?

Reply:
Would something like this work?

```js
/** @typedef {
  import('./typings').FILTransactionProcessing
} FILTransactionProcessing */

Reply:
it does! 🥳